fontc_crater/src/ci/html.rs

```diff
@@ -106,12 +106,24 @@ fn make_html(
             body {
                 h1 { "fontc_crater" }
                 div #explain {
-                    "compiling a large number of target fonts with both "
+                    "Compiling all known Google Fonts that have sources with both "
                     {a href = "https://github.com/googlefonts/fontc" { "fontc" } }
                     " and "
                     {a href = "https://github.com/googlefonts/fontmake" { "fontmake" } }
                     ", comparing the results."
                 }
+                div #explain {
```

Comment:
I would probably put this underneath the main table? It is closely related to the specific current run. Medium term, I think it would make sense to have an additional 'summary pane that just shows additional stats related to the latest run (and could include the # of matching families, for instance)

Reply:
Done